recipes/libavrocpp/all/conanfile.py

```diff
@@ -67,34 +52,42 @@ def source(self):
                 "find_package(Snappy)",
                 "",
             )
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
 
-    @property
-    def _source_subfolder(self):
-        return os.path.join("source_subfolder", "lang", "c++")
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "11")
 
     def requirements(self):
         self.requires("boost/[>=1.38.0]")
+
+    def build_requirements(self):
+        self.build_requires("boost/[>=1.38.0]")
```

Comment:
You should only add boost once, we do do not use version ranges in CCI (to ensure reproducible builds)